lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -93,6 +93,8 @@ jinja2==3.1.6
 jmespath==1.0.1
 jsonref==1.1.0
 jsonschema==4.25.0
+json-stream==2.3.3
+json-stream-rs-tokenizer==0.4.29
```

Comment:
Not needed? See note in readme https://github.com/smheidrich/py-json-stream-rs-tokenizer

Reply:
I've removed the dependency on `json-stream-rs-tokenizer`